masonry/src/widget/widget_state.rs

```diff
@@ -148,20 +148,20 @@ impl WidgetState {
             is_explicitly_disabled: false,
             baseline_offset: 0.0,
             is_hot: false,
-            needs_layout: false,
-            needs_paint: false,
-            needs_accessibility_update: false,
-            needs_window_origin: false,
+            needs_layout: true,
+            needs_paint: true,
+            needs_accessibility_update: true,
+            needs_window_origin: true,
             has_focus: false,
             request_anim: false,
```

Comment:
should this also be:
```suggestion
            request_anim: true,
```
?
I see that this is set in the `Spinner` widget at `WidgetAdded`. Or should this explicitly be set somewhere else for widgets needing animation directly on creation (like the `Spinner`)?

Reply:
No, I think you're right. Having all the machinery run once by default is easier. Fewer footguns for users that way.